server/pages/index.js

```diff
@@ -525,6 +525,13 @@ module.exports = function() {
 		} );
 	} );
 
+	app.get( '/support-user', function( req, res ) {
```

Comment:
We'll probably want to do a login check, and 404 otherwise. If it's possible doing an a8c check would also be ace.

Reply:
Good point, I'll look into it. :+1:

Reply:
After some experimenting with this I think we can't add login check here for the same reason that we had to move this logic from sections to separate route. Rendering a `404` here would again load 3rd party resources, so I think we must _always_ expose this route and run the code in pug template no matter what.

I did attempt a workaround with moving this logic to pug template, but it might be weird.

Reply:
We'd want to at least do a basic cookie check, as I wouldn't want bots crawling to this section.

Reply:
Note, that we're also missing setting X-FRAME-OPTION headers here. @vindl do you mind spinning up just a PR for the new express section? We'll want to land this first without disabling SU functionality. I can help if you're having trouble with the auth check.

Reply:
Ok, noting that work on this part will be split out and continued in https://github.com/Automattic/wp-calypso/pull/21984.